static/app/views/dashboardsV2/widgetBuilder/buildSteps/filterResultsStep/releaseSearchBar.tsx

```diff
@@ -67,6 +68,13 @@ export function ReleaseSearchBar({
     );
   }
 
+  const releaseSearchValidator: SearchValidationFunction = (_filter, key, _value) => {
```

Comment:
Since SmartSearchBar already knows which tags should be supported, another way of going about this is either 1) changing the default behavior to highlight invalid keys or 2) adding a prop like `allowInvalidKeys`. Just seems to me like defining the valid keys in two separate locations is asking for trouble

Reply:
Hey Malachi, thanks for the feedback! Good point, I actually didn't realize we were already passing supportedTags to the smartSearchBar. I've refactored this pr a bit, to allow smartSearchBar to specify whether or not to highlight invalid tags (showUnsupportedTags prop). This way, we won't have two separate areas where we're managing valid release health keys like you mentioned